styles/designs/cardboard/parts/_exercises-components.scss

```diff
@@ -34,8 +42,33 @@ $InjectedExercise__Question__Wrapper: (
     )
 );
 
-$Exercise__Problem__Container: exerciseChildContainer((child: problem, name: "ProblemContainer", hasImageTableOnTop: false, hasInlineListOnTop: false));
-$Exercise__Problem__Container--WithImageTableOnTop: exerciseChildContainer((child: problem, name: "ProblemContainerWithImageTableOnTop", hasImageTableOnTop: true));
+$Exercise__Problem__Container: (
+    _name: "ProblemContainer",
+    _subselector: " > .os-problem-container",
+    _properties:(
+      display: table-cell,
+      width: 100%
+    )
+);
+
+$Exercise__Problem__Container--WithFirstElement: (
+    _name: "ProblemContainerWithFirstElement",
+    _subselector: " > .os-problem-container.has-first-element",
+    _properties:(
+      display: table-cell,
+      vertical-align: bottom,
+      padding-top: v-spacing(1),
+      width: 100%,
+    )
+);
+
+$Exercise__Problem__Container--FromMultipartQuestion: (
+    _name: "ProblemContainerFromMultipartQuestion",
+    _subselector: " > .os-problem-container",
+    _properties:(
+        display: table,
+    )
+);
```

Comment:
Similar as in Carnival, I created new components that are not using `exerciseChildContainer` function. We want to delete it within Baked Styles refactoring. In new components we are using direct children as selectors, because problem container appears twice in multipart questions, and has different properties for each of the instance.